k8s/zookeeper/apptest/tester/Dockerfile

```diff
@@ -15,8 +15,9 @@ RUN set -eu \
 RUN set -eu \
     && wget -q -O /bin/kubectl https://storage.googleapis.com/kubernetes-release/release/v1.12.0/bin/linux/amd64/kubectl \
     && chmod 755 /bin/kubectl \
-    && wget -q -O /opt/zookeeper-3.4.14.tar.gz https://www-eu.apache.org/dist/zookeeper/zookeeper-3.4.14/zookeeper-3.4.14.tar.gz \
-    && tar -xvf /opt/zookeeper-3.4.14.tar.gz
+    && wget -q -O /opt/zookeeper-3.6.1.tar.gz https://downloads.apache.org/zookeeper/zookeeper-3.6.1/apache-zookeeper-3.6.1-bin.tar.gz \
```

Comment:
3.5 ?

Reply:
It is not necessary, as Zookeeper 3.6.1 client are compatible with 3.5 servers.
https://zookeeper.apache.org/releases.html#releasenotes